src/scripts/apiScripts.js

```diff
@@ -0,0 +1,50 @@
+import cookie from 'react-cookies'
+
+export const BACK_AVAILABLE = ( process.env.REACT_APP_BACK_API_AVAILABLE == 'true' ? true : false ); 
+export const API_KEY = process.env.REACT_APP_BACK_API_KEY;
+export const BACK_HOST_NAME = process.env.REACT_APP_BACK_API_HOST_NAME;
+                             
+export const loginAPI = async (username,password) => {
+  const query_string='?username='+username+'&password='+password;
```

Comment:
nit: this can also be declared likes this and can be read easier.
```suggestion
  const query_string=`?username=${username}&password=${password}`;
```